django/db/models/base.py

```diff
@@ -245,10 +252,13 @@ def __new__(cls, name, bases, attrs):
                     field = None
                 new_class._meta.parents[base] = field
             else:
-                # .. and abstract ones.
+                # add fields from abstract base class if it wasn't overridden
```

Comment:
It seems most other comments capitalize the first letter and use a period at the end.